docs/User-Guide_Fine-Tuning.md

```diff
@@ -35,7 +35,34 @@
 	# disp.screen0_output_mode=1280x720p60
 
 	mkimage -C none -A arm -T script -d /boot/boot.cmd /boot/boot.scr
-
+	
+### Screen resolution within Xorg <sub>[Thx @maxlinux2000](https://forum.armbian.com/topic/10403-add-undetected-hdmi-resolution-to-x11xorg/)</sub>
```

Comment:
Perhaps also this https://forum.armbian.com/topic/9368-orangepi-3-h6-allwiner-chip/?do=findComment&comment=100127 how to set via boot parameter - but I didn't test this.

Reply:
Me neither, that is why I did not add (yet)

Reply:
OK, then once its tested.